client/controller/index.web.js

```diff
@@ -172,3 +174,16 @@ export const notFound = ( context, next ) => {
 
 	next();
 };
+
+export function selectSiteIfLoggedIn( context, next ) {
```

Comment:
Is there any chance this could not live in the primary controller, but in the themes one for example? It caused a huge (over 20%) inflation on the main entry point.

cc @Automattic/team-calypso-frameworks @sgomes for awareness.

Reply:
Yes, site selection is specific for `my-sites` and should be at least in `my-sites/controller`, never in the top-level `calypso/controller`. Or it could be in `my-sites/themes/controller`.

Reply:
@tyxla thanks for bringing it up! @sgomes already submitted a fix, I will be reviewing it